### ISSUE TITLE
Accessibility: Insufficient contrast on Agenda links

### ISSUE THREAD

**Issue by robcrowley** [dddwa/dddperth-website#51]
Insufficient contrast, ratio is only 1.6:1 and increases to 2.1:1 on hover. Users who are colour-blind or have low vision will find them difficult to read.

**Recommendation:** choose a darker `background-color` which provides a contrast ratio of at least 4.5:1. The green used in the DDD header section meets this requirement if we don't want to introduce new colours.

**Comment by robdmoore** [comment]
Is this now fixed? cc @Macca2805, @robcrowley, @matthewjward

**Comment by Macca2805** [comment]
Yes this should now be resolved.
